Tests/ContentfulPersistenceTests/TestModels/Post+CoreDataProperties.swift

```diff
@@ -27,14 +28,15 @@ extension Post: EntryPersistable {
     @NSManaged var slug: String?
     @NSManaged var tags: Data?
     @NSManaged var title: String?
-    @NSManaged var author: NSOrderedSet?
+    @NSManaged var authors: NSOrderedSet?
     @NSManaged var category: NSOrderedSet?
     @NSManaged var theFeaturedImage: Asset?
 
-    static func mapping() -> [FieldName: String] {
+    static func fieldMapping() -> [FieldName: String] {
         return [
             "title": "title",
-            "featuredImage": "theFeaturedImage"
+            "featuredImage": "theFeaturedImage",
+            "author": "authors"
```

Comment:
Hmm, should the type of `author` also be adopted in the recorded json, or do you want to demonstrate here renaming of a json field in swift? (If so, maybe we can come up with something more speaking, aka calling the swift field `authorSet` or something?)